src/lib/import-export/export/export-manager.ts

```diff
@@ -18,6 +18,9 @@ export async function exportBackup(
 				removeExportListeners();
 			}
 			break;
+		} else {
+			onEvent( { event: ExportEvents.EXPORT_ERROR, data: null } );
+			throw new Error( 'No suitable exporter found for the site' );
```

Comment:
I noticed that we weren't raising an error when no exporter are found.

Reply:
Thanks for adding this, I had a similar concern earlier in https://github.com/Automattic/studio/pull/391#discussion_r1695249186 👍